app/geo/gui/src/ts/twxplore/gui/geo/components/filterSliders/FilterSliders.tsx

```diff
@@ -33,7 +34,9 @@ const FilterSlidersImpl: React.FunctionComponent<{featureType: string}> = ({
     (rootState: RootState) => rootState.app.map
   );
 
-  const featureTypeFilter = state.featuresByType[featureType].attributesState;
+  const featureTypeState = state.featuresByType[featureType].featureTypeState;
+  const featureTypeAttributeState =
```

Comment:
Nit: attributesState vs. AttributeState?

Reply:
I actually renamed this to attributeStates, because it is essentially a dictionary of several 'MapFeatureAttributeState's. 
Where key is attributeName and value is the MapFeatureAttributeState
.attributeStates[frequency] = {min: 0, max:100, idx: 5}